avm/res/batch/batch-account/main.bicep

```diff
@@ -74,61 +74,89 @@ param customerManagedKey customerManagedKeyType
 @description('Optional. The managed identity definition for this resource.')
 param managedIdentities managedIdentitiesType
 
-var formattedUserAssignedIdentities = reduce(map((managedIdentities.?userAssignedResourceIds ?? []), (id) => { '${id}': {} }), {}, (cur, next) => union(cur, next)) // Converts the flat array to an object like { '${id1}': {}, '${id2}': {} }
-var identity = !empty(managedIdentities) ? {
-  type: (managedIdentities.?systemAssigned ?? false) ? 'SystemAssigned' : (!empty(managedIdentities.?userAssignedResourceIds ?? {}) ? 'UserAssigned' : 'None')
-  userAssignedIdentities: !empty(formattedUserAssignedIdentities) ? formattedUserAssignedIdentities : null
-} : null
-
-var accountAccessNetworkProfileIpRules = [for allowedIpRule in networkProfile.?accountAccess.?allowedIpRules ?? []: {
-  action: 'Allow'
-  value: allowedIpRule
-}]
-
-var nodeManagementAccessNetworkProfileIpRules = [for allowedIpRule in networkProfile.?nodeManagementAccess.?allowedIpRules ?? []: {
-  action: 'Allow'
-  value: allowedIpRule
-}]
+var formattedUserAssignedIdentities = reduce(
```

Comment:
I'd kindly ask to merge #1481 first, to simplify the review of this PR and correctly track the modules still missing v2 formatting

Reply:
Hey @eriqua, I actually already reviewed this PR including the reformatting (which I tihnk applie to like 8-10 modules), wasn't too bad :)

Reply:
Never doubted that 😄 The problem is it did not reformat the whole module but only the main template and would have left us with a mixed up state. 
Thanks for approving the PR, now we are left with 5 missing formatting that I asked to be included as part of the open MR. We need to keep an eye on 2 PRs not yet opened. Storage account and flexible servers. You can resolve this one

Reply:
You shall resolve it :P

Reply:
And resolved it is

Reply:
Much appreciated dear author :D